test/e2e/e2e_test.go

```diff
@@ -229,10 +237,16 @@ var _ = Describe("controller", Ordered, func() {
 								{
 									Name:        hashringName,
 									StorageSize: "100Mi",
+									CommonFields: v1alpha1.CommonFields{
+										Version: getThanosVersion(),
+									},
 								},
 								{
 									Name:        hashringTwoName,
 									StorageSize: "100Mi",
+									CommonFields: v1alpha1.CommonFields{
+										Version: getThanosVersion(),
+									},
```

Comment:
This starts a router too right @philipgough? If so, will need to add config for it here, for version changes